### DISCUSSION TITLE
How did you come across Lighty?

### DISCUSSION THREAD

**Discussion by andi-makes** [SchmarrnDevs/Lighty#12] [Polls]
I never really told anyone about it, I just uploaded it to Modrinth - so I guess most people just found it on Modrinth, but I would love to hear if someone did find Lighty through other means.

**Comment by ichigochantf2** [comment]
Found it on prism launchers mod selection screen which does use modrinth but yeah

**Comment by SlightlyModded1089** [comment]
Found this mod on Modrinth while searching for a light overlay mod compatible with Exordium (an FPS-reducing mod for GUI in order to free up CPU bandwidth for world rendering). Will update after testing!

**Comment by andi-makes** [comment]
Thinking about it, I only link from the modrinth page to this poll, so it is kinda misrepresenting everything a bit?

Anyways, thanks to everybody who voted, I'll close this discussion for now. Maybe I will do a better thought through poll another time, who knows.

Have fun!
